this patch fixes problem that /user/password generates 404 not found page

--- ./mass_url.module	2004-11-24 17:00:33.000000000 +0200
+++ /tmp/mass_url.module	2005-12-06 17:30:16.000000000 +0200
@@ -59,7 +61,7 @@
     return;
   }
   else if ($url_username == ('password')) {
-    user_page();
+    user_pass();
     return;
   }
   else if ($url_username == ('help')) {
